packages/plugins/typed-variable-modal/src/TypedVariableModal.js

```diff
@@ -238,7 +238,7 @@ export class TypedVariableModal extends Modal {
   renderContent_(contentContainer) {
     const varNameContainer = this.createVarNameContainer_();
     this.variableNameInput_ = varNameContainer
-        .querySelector('.typed-modal-variable-name-input');
+        .querySelector('.typedModalVariableNameInput');
```

Comment:
Just checking, but this change is so that we are following pattern seen in Blockly core right?